### PR TITLE
hamburger menu created inapp.components.html, menu added to home.page…

### PR DESCRIPTION
**Pull request by Jankita58**
…, it shows username

### CODE REVIEW

File: sk0-front/src/app/app.component.ts [hunk 1]
```diff
@@ -9,5 +10,9 @@ import { IonicModule } from '@ionic/angular';
   imports: [IonicModule],
 })
 export class AppComponent {
-  constructor() {}
+  constructor(private authService: AuthService) {}
+
+  get userName() {
+    return this.authService.userName;
+  }
```
Comment (rolcho) [inline sk0-front/src/app/app.component.ts]: I think it is different in the dev bev branch. There should be a BehavorSubject in the dev branch.

```suggestion
  userName$ = this.authService.userName
  
  constructor(private authService: AuthService) {}
```

File: sk0-front/src/app/app.component.html [hunk 2]
```diff
@@ -1,3 +1,15 @@
 <ion-app>
-  <ion-router-outlet></ion-router-outlet>
+  <ion-menu contentId="main-content">
+    <ion-header>
+      <ion-toolbar>
+        <ion-title>
+          {{ userName }}
```
Comment (rolcho) [inline sk0-front/src/app/app.component.html]:
```suggestion
          {{ userName$ | async }}
```

**Review by rolcho — CHANGES_REQUESTED** [review]
Looking good, the missing part is to merge origin/dev into your branch

**Review by rolcho — APPROVED** [review]
Looking good for me

**Review by markkovari — APPROVED** [review]

**Review by bendesbeus — APPROVED** [review]

**Pull request merged**